seatunnel-e2e/seatunnel-flink-new-connector-e2e/src/test/java/org/apache/seatunnel/e2e/flink/oracle/FakeSourceToOracleIT.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.e2e.flink.oracle;
+
+import org.apache.seatunnel.e2e.flink.FlinkContainer;
+import org.junit.Assert;
+import org.junit.Test;
+import org.testcontainers.containers.Container;
+
+import java.io.IOException;
+
+
+public class FakeSourceToOracleIT extends FlinkContainer {
+	@Test
+	@SuppressWarnings("magicnumber")
+	public void testFakeSourceToOracleSink() throws IOException, InterruptedException {
+		Container.ExecResult execResult = executeSeaTunnelFlinkJob("/oracle/fakesource_to_oracle.conf");
+		Assert.assertEquals(0, execResult.getExitCode());
```

Comment:
Need to test sink and source on flink and spark.
And need to check the data size and data type. Also need to test exactly-once mode if XA transaction are supported.